### DISCUSSION TITLE
Proposed Solution for MTA Server Login Attacks

### DISCUSSION THREAD

**Discussion by migferamiral** [multitheftauto/mtasa-blue#3747] [General]
Hello, for nearly 2 years, attackers have turned a system that MTA implemented for protection into an advantage for themselves, preventing players from logging into MTA servers. By sending thousands of requests using their real IP addresses, they can easily block server logins. When we enable the "Packet Tag" option, as shown in the video, the problem is resolved. MTA needs to enable this option by default for players to completely solve this issue.

Video: [https://youtu.be/FJfWUVAFGo0](https://youtu.be/FJfWUVAFGo0)

**Comment by sydefx1** [comment]
We support as Unicorn and Popcorn Roleplay.

**Comment by C4Ns-ART** [comment]
We are supporting @migferamiral 's Discussions and Solutions as Kızıl Gaming Community, Kızıl Roleplay
